Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -982,10 +999,51 @@ extension TelemetryWrapper {
         // MARK: Settings Menu
         case (.action, .open, .settingsMenuSetAsDefaultBrowser, _, _):
             GleanMetrics.SettingsMenu.setAsDefaultBrowserPressed.add()
-
+        case(.action, .open, .settingsMenuPasswords, _, _):
+            GleanMetrics.SettingsMenu.passwords.record()
         case(.action, .tap, .settingsMenuShowTour, _, _):
             GleanMetrics.SettingsMenu.showTourPressed.record()
 
+        // MARK: Logins and Passwords
+        case(.information, .emailLogin, .loginsPasswordDetected, _, _):
+            GleanMetrics.Logins.passwordDetected.record()
+        case(.action, .view, .loginsAutofillPromptShown, _, _):
+            GleanMetrics.Logins.autofillPromptShown.record()
+        case(.action, .close, .loginsAutofillPromptDismissed, _, _):
+            GleanMetrics.Logins.autofillPromptDismissed.record()
+        case(.action, .tap, .loginsAutofilled, _, _):
+            GleanMetrics.Logins.autofilled.record()
+        case(.action, .tap, .loginsAutofillFailed, _, _):
+            GleanMetrics.Logins.autofillFailed.record()
+        case(.action, .tap, .loginsManagementAddTapped, _, _):
+            GleanMetrics.Logins.managementAddTapped.record()
+        case(.action, .tap, .loginsManagementLoginsTapped, _, _):
+            GleanMetrics.Logins.managementLoginsTapped.record()
+        case(.action, .add, .loginsSaved, _, _):
+            GleanMetrics.Logins.saved.add()
+        case(.information, .foreground, .loginsSavedAll, _, let extras):
+            if let quantity = extras?[EventExtraKey.loginsQuantity.rawValue] as? Int64 {
+                GleanMetrics.Logins.savedAll.set(quantity)
+            } else {
+                recordUninstrumentedMetrics(category: category, method: method, object: object, value: value, extras: extras)
+            }
+        case(.action, .delete, .loginsDeleted, _, _):
+            GleanMetrics.Logins.deleted.add()
+        case(.action, .change, .loginsModified, _, _):
+            GleanMetrics.Logins.modified.add()
+        case(.action, .tap, .loginsSyncEnabled, _, let extras):
+            if let isEnabled = extras?[EventExtraKey.isLoginSyncEnabled.rawValue]
+                as? Bool {
```

Comment:
nit, feels strange to have this on another line
```suggestion
            if let isEnabled = extras?[EventExtraKey.isLoginSyncEnabled.rawValue] as? Bool {
```